### DISCUSSION TITLE
Analysis not yet started

### DISCUSSION THREAD

**Discussion by ITarrant** [matiasdelellis/facerecognition#600] [General]
Hi guys, I've installed the app, assigned memory and configured a model. But the app has not done anything in over a week. The only message I get is analysis not yet started - 834 images in queue.
![Screenshot from 2022-10-01 15-22-57](https://user-images.githubusercontent.com/36146295/193393978-48abd8c5-0c37-41dc-904d-9e926409af0e.png)
Anyone have any ideas what could be going on?

**Comment by ITarrant** [comment]
Seems the background job was not running. Manually starting it seems to have done something.

**Comment by matiasdelellis** [comment]
Hi @ITarrant 

The process is designed to run periodically, but it does not have any automatic execution. You must configure the schedule task, or run it manually. This is specially designed like this, to have precise control of its execution and the use of hardware resources.

See:
* https://github.com/matiasdelellis/facerecognition#schedule-background-job
* https://github.com/matiasdelellis/facerecognition/wiki/Schedule-Background-Task
